setup.py

```diff
@@ -40,6 +48,9 @@ def read(*parts):
         "gamsxcc",
         "optcc"
     ],
+    extras_require={
```

Comment:
Can we remove this part and add it as a requirement at the end of environment.yml